JavaFolder/code/src/main/java/xyz/krakenkat/app/interview/SubArrayWithGivenSum.java

```diff
@@ -0,0 +1,44 @@
+package xyz.krakenkat.app.interview;
+
+public class SubArrayWithGivenSum {
+
+	public static void main(String[] args) {
+
+		test(new int[] {
+				10,3,5,8,6,12,20,15,31
+		}, 31);
+		
+	}
+
+	
+	public static void test(int[] array, int num) {
```

Comment:
Remove the static method. The static method will cause issues in the unit tests.

Reply:
The developer made the promise to update the code. I trust him